src/main/java/com/ss/training/lms/controller/AdminBorrwerController.java

```diff
@@ -0,0 +1,118 @@
+package com.ss.training.lms.controller;
+
+import java.sql.SQLException;
+import java.util.List;
+
+import com.ss.training.lms.entity.Borrower;
+import com.ss.training.lms.service.admin.AdminBorrowerService;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class AdminBorrwerController {
+
+    @Autowired
+    AdminBorrowerService adminBorrowerService;
+
+    @PostMapping(path = "/lms/admin/borrowers/add")
+    public ResponseEntity<Borrower> addABorrower(@RequestBody Borrower borrower) {
+        HttpStatus status = HttpStatus.CREATED;
+
+        if (borrower == null || borrower.getCardNo() == null || borrower.getName() == null) {
+            status = HttpStatus.BAD_REQUEST;
+            return new ResponseEntity<Borrower>(borrower, status);
+        }
+        try {
+            adminBorrowerService.addABorrower(borrower);
+        } catch (SQLException e) {
+            e.printStackTrace();
+            status = HttpStatus.INTERNAL_SERVER_ERROR;
+            return new ResponseEntity<Borrower>(borrower, status);
+        }
+        return new ResponseEntity<Borrower>(borrower, status);
+    }
+    
+
+    @PostMapping(path = "/lms/admin/borrowers/delete")
+    public ResponseEntity<Borrower> deleteABorrower(@RequestBody Borrower borrower) {
+        HttpStatus status = HttpStatus.ACCEPTED;
+
+        if (borrower == null || borrower.getCardNo() == null || borrower.getName() == null) {
```

Comment:
We shouldn't check for a null name here. It could be that the borrower the user wants to delete has a null name.